__tests__/app.test.js

```diff
@@ -92,3 +92,88 @@ describe('GET /api/articles/:article_id', () => {
         })
     })
 })
+
+describe('GET /api/articles', () => {
+    test('status:200, responds with a JSON object containing an array of article objects, each of which contains the following properties: author, title, article_id, topic, created_at, votes, article_img_url', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+
+        const { articles } = body;
+
+        expect(articles).toHaveLength(5);
```

Comment:
As per my slack message, the reason you're getting the incorrect length is due to the type of 'JOIN' used in your model.